x-pack/legacy/plugins/rollup/__jest__/client_integration/job_create_review.test.js

```diff
@@ -4,7 +4,9 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { setupEnvironment, pageHelpers, nextTick } from './helpers';
+import { setupEnvironment, pageHelpers } from './helpers';
+import _ from 'lodash';
```

Comment:
nit: this could be `import { first } from 'lodash';` (in general we try to import just what we need)

Reply:
Sure, I guess it can be a bit of a trade-off. I find it's nice to have things namespaced (like lodash functions) which is kind of a kludge of utilities.

Another consideration is from a webpack/build perspective if we _only_ wanted to import certain functionality we could do [`import first from 'lodash/array/first';`](https://stackoverflow.com/questions/38171465/modules-import-and-destructuring-performances). But I think `lodash` is already being imported all over the place so bundling preformance won't be improved.

Will run with your suggestion in this case :)

Reply:
I recall there was an Kibana effort use webpack tree shaking, which is the basis of my suggestion - not sure if anything has been done on that front, though!